portal/server/api.js

```diff
@@ -856,9 +856,27 @@ const getPreviousUtilisationReportsByBank = async (token, bankId) => {
   }
 };
 
+const getLastestReportByBank = async (token, bankId) => {
+  if (!isValidBankId(bankId)) {
+    throw new Error(`Getting latest report failed - bank id '${bankId}' is invalid`);
+  }
+
+  const response = await axios({
+    method: 'get',
+    url: `${PORTAL_API_URL}/v1/banks/${bankId}/utilisation-reports/latest`,
+    headers: {
+      Authorization: token,
+      'Content-Type': 'application/json',
+    },
+  });
+
+  return response.data;
+};
+
 const getDueReportDatesByBank = async (token, bankId) => {
   if (!isValidBankId(bankId)) {
-    throw new Error(`Getting due utilisation reports failed for id ${bankId}`);
+    console.error('Getting due utilisation reports failed for id %s', bankId);
+    return false;
```

Comment:
What was the reason for this change?

Reply:
I think this was something that went wrong during a rebase onto `main-stbr`